Analysis/R/prepare_stan_input.R

```diff
@@ -477,6 +540,30 @@ prepare_stan_input <- function(
   # Add scale of prior on the sd of regression coefficients
   stan_data$beta_sigma_scale <- config$beta_sigma_scale
   
+  # Priors for intercept
+  stan_data$mu_alpha <- 0
```

Comment:
I'm assuming these are temporary? Not sure otherwise why priors would be set here

Reply:
The priors for alpha are now also par of the config, but we still need to define the check functions.